www/include/configuration/configObject/contact/formContact.ihtml

```diff
@@ -83,10 +83,15 @@
         {if $o == "a" || $o == "c"}
             <tr class="list_lvl_1"><td class="ListColLvl2_name" colspan="2">{$form.required._note}</td></tr>
         {/if}
+        {if is_array($modules_template)}
+            {foreach item=module_template from=$modules_template}
+                {include file=$module_template}
+            {/foreach}
+        {/if}
     </table>
     </div>
     <div id='tab2' class='tab'>
-    <table class="formTable table">
+        <table class="formTable table">
```

Comment:
if you correct this indentation, you should do the same for all the lines until the next `</table>` tag